src/Symfony/Component/VarDumper/Cloner/VarCloner.php

```diff
@@ -128,6 +128,15 @@ protected function doClone(mixed $var): array
                             $stub->value = $v;
                             $stub->handle = $h;
                             $a = $this->castObject($stub, 0 < $i);
+                            if ($stub->value !== '') {
+                                $uninitializedProps = array_diff_key(get_class_vars($stub->value::class), $a);
+                                $stubClass = $stub->value::class;
+                                $stub->attr['uninitialized'] = [];
+                                foreach ($uninitializedProps as $key => $value) {
+                                    $rp = new \ReflectionProperty($stubClass, $key);
+                                    $stub->attr['uninitialized'][] = [$key => $rp->getType()->getName()];
```

Comment:
why using a list of single-item arrays with an unknown key for this `$stub->attr['uninitialized']` ? why not using `$stub->attr['uninitialized'][$key] = $type` here ?